<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Intake</class>
 <widget class="QMainWindow" name="Intake">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intake</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SUPER DOPE LOGO</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="menuIntake">
    <property name="title">
     <string>Intake</string>
    </property>
    <addaction name="newBag"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuIntake"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newBag">
   <property name="text">
    <string>New Bag</string>
   </property>
  </action>
  <action name="checkInventory">
   <property name="text">
    <string>Check Inventory</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
